### PR TITLE
Disabled movement while adjusting dribbler value with spacenav

### LINKED ISSUE #440: Disable other motion when adjusting dribbler speed with 3d mouse
As it is, it's hard to adjust the dribbler without making the robot move.

### PR DESCRIPTION
**Pull request by chachmu**
Added spacenav joystick dribbler deadzone values to the config tab and stopped movement while adjusting dribbler setting. Closes #440

### CODE REVIEW

File: soccer/joystick/SpaceNavJoystick.cpp [hunk 1]
```diff
@@ -34,50 +39,55 @@ void SpaceNavJoystick::update() {
...
-                _controlValues.rotation = 0;
-            }
-
-            //  the y axis corresponds to pulling the knob up and down, which we
-            //  use to control the dribbler
-            //  moving the knob in other directions unintentionally moves the y
-            //  axis, so we give it a large deadzone
-            const int DribblerAxisDeadzone = 200;
-
             //  we don't want to increment it at each iteration, otherwise it
```
Comment (justbuchanan) [inline soccer/joystick/SpaceNavJoystick.cpp]: Let's move this comment down a bit so it's right above the timing it describes: `(now - _lastDribbleTime)`.

File: soccer/joystick/SpaceNavJoystick.cpp [hunk 2]
```diff
@@ -34,50 +39,55 @@ void SpaceNavJoystick::update() {
...
+                    if (_controlValues.dribblerPower < 0)
+                        _controlValues.dribblerPower = 0;
+                    if (_controlValues.dribblerPower > 1)
+                        _controlValues.dribblerPower = 1;
+                }
 
-            if ((abs<int>(sev.motion.y) > DribblerAxisDeadzone) &&
-                ((now - _lastDribbleTime) > DribbleStepTime)) {
-                _lastDribbleTime = now;
+                _controlValues.translation = Geometry2d::Point(0, 0);
```
Comment (justbuchanan) [inline soccer/joystick/SpaceNavJoystick.cpp]: It would probably be good to zero out the rotation value here too.

**Comment by justbuchanan** [comment]
I looked through this and made a couple notes, but overall looks good!  It's failing the style check right now, so be sure to run the code formatter.  If you click on the "Details" link next to the "circle/style" check in the list, it will give you a command to run to fix all formatting.

**Comment by jgkamat** [comment]
https://circle-artifacts.com/gh/RoboJackets/robocup-software/1191/artifacts/0/tmp/circle-artifacts.9FepuGh/clean.patch

**Pull request merged**